src/components/Book/BookItem.jsx

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+const BookItem = (props) => {
+  const { book } = props;
+  return (
+    <div className="book-item">
+      <div>
+        <div className="category">{book.category}</div>
+        <div className="title">{book.title}</div>
+        <div className="author">{book.author}</div>
+        <ul className="actions">
+          <li><Link to="/">Comments</Link></li>
+          <li><Link to="/">Remove</Link></li>
```

Comment:
- [ ] I think it would be better if the `remove` is in the `<button>` tag because is a button and you will add soon the functionality to that button to remove book to the list 😁